modules/rosa-create-objects.adoc

```diff
@@ -717,6 +717,9 @@ a|--cluster <cluster_name>\|<cluster_id>
 |--replicas
 |Required when autoscaling is not configured. The number (integer) of machines for this machine pool.
 
+|--tags                            
+|Apply user defined tags to all resources created by ROSA in AWS. Tags are comma separated, for example: 'key value, foo bar'.
```

Comment:
I think these are supposed to be backticks since they don't resolve properly in the rendered output:
```suggestion
|Apply user defined tags to all resources created by ROSA in AWS. Tags are comma separated, for example: `key value, foo bar`.
```
If the single apostrophe is important to the syntax, maybe you could include both, like this:
```suggestion
|Apply user defined tags to all resources created by ROSA in AWS. Tags are comma separated, for example: `'key value, foo bar'`.
```

Either way it's up to you, I just feel like this should be wrapped in backticks if it refers to something that shows up in a CLI or file.